### PR TITLE
add ability to prefer .slime over .slim for generated files

### PR DESCRIPTION
**Pull request by hedgesky**
It could be done with following line in config:

```elixir
config :phoenix_slime, :use_slime_extension, true
```

After that all generated files will have `.slime` extension instead of `.slim`.

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @hedgesky!  This is a great addition.  I'll look this over today and let you know if I have any feedback 👍

**Comment by doomspork** [comment]
Sorry for the maddening delay @hedgesky, life got crazy to say the least (although that's no excuse).

I think this is good to go but I do have a thought:  Given that we aren't going to implement slim 100% instead favoring Elixirism where appropriate, should we default to `.slime` instead of `.slim`?

**Comment by hedgesky** [comment]
Thanks for assessing this!
I agree; made corresponding changes in a separate commit so one could easily revert it if the need occurs. The only thing that bothers me is that this is a breaking change. I see to options here:

- Bump version to `1.0.0` accordingly to semver;
- Don't change major version because we haven't reached `1.0.0` yet and we can afford breaking changes.

@doomspork, what do you think?

**Comment by doomspork** [comment]
@hedgesky let's call it `1.0.0`, I'd rather not break a bunch of apps 😀

**Comment by hedgesky** [comment]
Bumped version and rebased this onto fresh master to avoid conflicts. @doomspork, are we good to go?

File: mix.exs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 defmodule PhoenixSlime.Mixfile do
   use Mix.Project
 
-  @version "0.10.0"
+  @version "1.0.0"
```
Comment (hedgesky) [inline mix.exs:4]: 🎉

**Comment by doomspork** [comment]
Thank you @hedgesky!

**Pull request merged**